.qt6/vcpkg.json

```diff
@@ -38,11 +38,6 @@
       "default-features": false
     },
     "nu-book-zxing-cpp",
-    "spix-qt6",
-    {
-      "name": "sentry-native",
```

Comment:
Sentry also?

Reply:
@m-kuhn , oups -- typo, but actually it had to be disabled in system_qt6 too, so maybe let's no re-trigger a build here and try to re-enable it in system_qt6 (or a separate PR altogether after the main qt6 work is done)?

Reply:
(i.e. I suspect we'll need to update sentry to 0.6.X, which will be its own interesting struggle :wink:)